### DISCUSSION TITLE
Taiko on osu!lazer experiences consistent stutters and a graphical issue

### DISCUSSION THREAD

**Discussion by christianlewis21** [ppy/osu#20023] [Q&A]
### Type

Performance

### Bug description

I was playing osu!lazer and tried to play taiko. When playing taiko, the notes will consistently stutter. The sides of the notes are also slightly glitchy. They will flash very fast. This is not visible on the video (I don't think). Neither of these things happen on osu!stable.

My specs should be high enough and my drivers are updated.
Could also not be an issue with a game and just that I'm not very good at taiko.

### Screenshots or videos

https://user-images.githubusercontent.com/73616729/187332530-fe2444a2-b3a2-4935-87b0-a9592a6730b2.mp4

Much Slower Diff:

https://user-images.githubusercontent.com/73616729/187332535-491905b1-a91c-4849-bc65-a4fa11350e10.mp4

With Skin:

https://user-images.githubusercontent.com/73616729/187334086-1e0f2998-8474-491f-92f0-8041da48c9ef.mp4

Much Faster Diff:

https://user-images.githubusercontent.com/73616729/187334669-93ba0285-1194-4a08-b69f-5b1bf1f6561f.mp4

Zoomed In:

https://user-images.githubusercontent.com/73616729/187336087-46091a38-bfe5-4aee-bc8f-ffc58711eb0d.mp4

### Version

2022.821.0

### Logs

[database.log](https://github.com/ppy/osu/files/9449008/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9449009/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9449010/network.log)
[performance.log](https://github.com/ppy/osu/files/9449011/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9449012/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9449013/updater.log)

**Comment by peppy** [comment]
This is probably your monitor (ghosting or otherwise). It happens when objects scroll right to left on LCD displays.

Also your videos are recorded at 30fps and aren't that helpful.

**Comment by christianlewis21** [comment]
ok thank you
